lib/ueberauth/strategy/auth0/oauth.ex

```diff
@@ -14,8 +14,10 @@ defmodule Ueberauth.Strategy.Auth0.OAuth do
         client_secret: {:system, "AUTH0_CLIENT_SECRET"}
   """
   use OAuth2.Strategy
+  alias OAuth2.Client
+  alias OAuth2.Strategy.AuthCode
 
-  def options() do
+  def options do
```

Comment:
This is not a complain. I just dislike Credo's warning about this. A function, even with zero arity, should always have parentheses to indicate it is a function.